File: v2/test_files/test2.py
# Estructura anidada

# Inicialización de variables
contador_externo = 0
limite_externo = 4
limite_interno = 6
acumulador = 0.0
estado = True

# Bucle while externo
while contador_externo < limite_externo:
    # Condición if dentro del while
    if estado:    
        # Condición if dentro del if
        if acumulador == 0.0:
            for indice_interno in range(limite_interno):
                acumulador = 1.5 + acumulador 
            # dedentación
            estado = False
            mensaje = "El acumulador ha sido actualizado"
        # dedentación
    # Incrementar la variable del bucle while
    contador_externo = 1 + acumulador 
# Resultado final almacenado en una variable
resultado_final = acumulador # Siempre hacer un ENTER despues de escribir la ultima linea!!

File: v2/output_files/test2.c
#include <stdio.h>

int main() {

int contador_externo = 0;
int limite_externo = 4;
int limite_interno = 6;
float acumulador = 0.0;
bool estado = true;
while (contador_externo < limite_externo) {
	if(estado) {
	if(acumulador == 0.0) {
	for (int indice_interno = 0; indice_interno <  limite_interno; indice_interno++) {
	acumulador = 1.5 + acumulador;
}
estado = false;
char mensaje[] = "El acumulador ha sido actualizado";
}
}
contador_externo = 1 + acumulador;
}
int resultado_final = acumulador;

return 0;
}
